django/db/backends/postgresql/base.py

```diff
@@ -153,6 +154,13 @@ class DatabaseWrapper(BaseDatabaseWrapper):
     # PostgreSQL backend-specific attributes.
     _named_cursor_idx = 0
 
+    def get_database_version(self):
+        """
+        Return a tuple of the database's version.
+        E.g. for pg_version 120004, return (12, 0, 4).
+        """
+        return tuple(map(int, wrap(str(self.pg_version), 2)))
```

Comment:
This is not truth for PostgreSQL > 10, see [docs](https://www.postgresql.org/docs/current/libpq-status.html#LIBPQ-PQSERVERVERSION):

_"Applications might use this function to determine the version of the database server they are connected to. The result is formed by multiplying the server's major version number by 10000 and adding the minor version number. For example, version 10.1 will be returned as 100001, and version 11.0 will be returned as 110000."_

Reply:
Prior to major version 10, PostgreSQL used three-part version numbers, but we don't support PostgreSQL < 10 anymore so I don't think it's worth adding a workaround, e.g.
```diff
diff --git a/django/db/backends/postgresql/base.py b/django/db/backends/postgresql/base.py
index 630da22964..6408b8f4e5 100644
--- a/django/db/backends/postgresql/base.py
+++ b/django/db/backends/postgresql/base.py
@@ -158,7 +158,10 @@ class DatabaseWrapper(BaseDatabaseWrapper):
         Return a tuple of the database's version.
         E.g. for pg_version 120004, return (12, 4).
         """
-        return divmod(self.pg_version, 10000)
+        major, minor = divmod(self.pg_version, 10000)
+        if major < 10:
+            return (major, *divmod(minor, 100))
+        return (major, minor)
 
     def get_connection_params(self):
         settings_dict = self.settings_dict
diff --git a/tests/backends/postgresql/tests.py b/tests/backends/postgresql/tests.py
index ce8ed7b4d5..f8686f6f99 100644
--- a/tests/backends/postgresql/tests.py
+++ b/tests/backends/postgresql/tests.py
@@ -314,6 +314,8 @@ class Tests(TestCase):
         new_connection = connection.copy()
         new_connection.pg_version = 110009
         self.assertEqual(new_connection.get_database_version(), (11, 9))
+        new_connection.pg_version = 90105
+        self.assertEqual(new_connection.get_database_version(), (9, 1, 5))
 
     @mock.patch.object(connection, "get_database_version", return_value=(9, 6))
     def test_check_database_version_supported(self, mocked_get_database_version):
```